src/Rules.Framework/Builder/RulesEngineOptionsValidator.cs

```diff
@@ -29,6 +29,23 @@ public static void EnsureValid(RulesEngineOptions rulesEngineOptions)
                 rulesEngineOptions.DataTypeDefaults,
```

Comment:
Github does not let me add a coment to the line 9 of this file 😢 

suggestion: Rename the method EnsureValid() to Validate().